src/app/snapp_test_transaction/lib/commands.ml

```diff
@@ -354,9 +354,13 @@ module Events = struct
 end
 
 module Util = struct
-  let keypair_of_file =
-    Secrets.Keypair.Terminal_stdin.read_exn ~should_prompt_user:false
-      ~which:"payment keypair"
+  let keypair_of_file ?(which = "Fee Payer") f =
+    printf "%s keyfile\n" which ;
+    Secrets.Keypair.Terminal_stdin.read_exn ~which f
+
+  let snapp_keypair_of_file ?(which = "Snapp Account") f =
```

Comment:
Nit: `let snapp_keypair_of_file ?(which = "Snapp account") f = keypair_of_file ~which f`. No need to repeat the logic.